neuro_admin_client/__init__.py

```diff
@@ -471,15 +471,15 @@ async def update_org_defaults(
         self,
         org_name: str,
         user_default_credits: Decimal | None,
-        notification_balance_depletion_seconds: int | None = None,
+        depletion_intervals: list[int] | None = None,
```

Comment:
To keep the backward compatibility the type should be `list[int]|int|None` where `int` should be processed as a list of one element probably.

Reply:
@asvetlov it's actually a very new property, and it was not used yet by any other services/libs. So basically  there is nothing to do here in terms of a backward compatibility